packages/image_picker/image_picker_for_web/example/integration_test/image_picker_for_web_test.dart

```diff
@@ -1,27 +1,30 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+// ignore_for_file: always_specify_types
+
 
 import 'dart:convert';
-import 'dart:html' as html;
-import 'dart:typed_data';
+import 'dart:js_interop';
 
 import 'package:flutter_test/flutter_test.dart';
 import 'package:image_picker_for_web/image_picker_for_web.dart';
 import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
 import 'package:integration_test/integration_test.dart';
+import 'package:web/web.dart' as web;
 
 const String expectedStringContents = 'Hello, world!';
 const String otherStringContents = 'Hello again, world!';
-final Uint8List bytes = const Utf8Encoder().convert(expectedStringContents);
-final Uint8List otherBytes = const Utf8Encoder().convert(otherStringContents);
-final Map<String, dynamic> options = <String, dynamic>{
-  'type': 'text/plain',
-  'lastModified': DateTime.utc(2017, 12, 13).millisecondsSinceEpoch,
-};
-final html.File textFile = html.File(<Uint8List>[bytes], 'hello.txt', options);
-final html.File secondTextFile =
-    html.File(<Uint8List>[otherBytes], 'secondFile.txt');
+final JSArray bytes =
```

Comment:
Individual temporary suppressions should never be made at the file level, because it allows new, unrelated violations.

See https://github.com/flutter/packages/blob/bb97da8ec627466fdb45cdd36e2c0e35dee40e9e/packages/google_sign_in/google_sign_in_web/lib/src/flexible_size_html_element_view.dart#L76-L78 for the correct suppression for this particular issue.

Reply:
i tried that too - https://github.com/flutter/packages/pull/5799/commits/ccf7796cb264b37e1a38f5e1833e0642dbd90eed
it was still failing. trying again

Reply:
failed again @stuartmorgan

Reply:
i had to add 4 (2 for each declaration) ignore to make this work

Reply:
`// ignore` operates on lines, not statements, so that's expected for a multi-line statement where each line has a violation.

Reply:
These declarations will need explanatory TODOs, as in the example I linked to. We should never be ignoring lint rules without explaining why.

Reply:
Added todo's under your name @ditman because i am not a team member :P

Reply:
this todo is not needed anymore.